extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -103,11 +106,15 @@ export async function createAzureFunction(connectionString: string, schema: stri
 			// check for the new function file to be created and dispose of the file system watcher
 			const timeoutForFunctionFile = utils.timeoutPromise(constants.timeoutAzureFunctionFileError);
 			await Promise.race([newFunctionFileObject.filePromise, timeoutForFunctionFile]);
+			TelemetryReporter.createActionEvent(TelemetryViews.CreateAzureFunctionWithSqlBinding, TelemetryActions.finishCreateAzureFunctionWithSqlBinding)
```

Comment:
It would be good to be able to tie all events from the same flow execution together (so every time a user started the flow it would be a new "session"). Have you thought about how you're going to do that?

Reply:
I was thinking of creating a UUID for each session that would be passed in that we could then put in the additional properties but was concerned if that would be PII. Any thoughts?